packages/components/src/DateTimePickers/DateTime/Manager/Manager.component.js

```diff
@@ -22,19 +22,23 @@ function ContextualManager(props) {
 	const initialState = extractParts(props.value, getDateOptions());
 	const [state, setState] = useState(initialState);
 
+	function onChange(event, payload) {
+		if (props.onChange) {
+			const { datetime, textInput, errors, errorMessage } = payload;
+			props.onChange(event, { datetime, textInput, errors, errorMessage });
+		}
+	}
+
 	useEffect(() => {
 		if (props.value !== state.datetime) {
 			const nextState = extractParts(props.value, getDateOptions());
 			setState(nextState);
+			if (nextState.errors && nextState.errors.length > 0) {
+				onChange(null, nextState);
+			}
 		}
```

Comment:
lgtm

and, with triggering onchange here, won't even be  necessary to implement separate validation for "datetime" widget in form, since now it validates datetime as simple input text, so only date without time (not empty string) is considered as valid, and the form can be submitted, which is incorrect